packages/wasm/src/build.ts

```diff
@@ -18,6 +18,15 @@ import {
   witness as wasmWitness,
 } from '@penumbra-zone/wasm-bundler';
 
+export async function fetchBinaryFile(filename: string) {
```

Comment:
Fetch feels a bit like it's making a network call. Could we do something like `loadLocalBinary`?

Reply:
changed